script/sccache-helper.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+import lib.sccache as sccache
+
+def parse_args():
+  parser = argparse.ArgumentParser(description='Run sccache`')
+
+  parser.add_argument('--azure-container',
+                      help='Name of azure container to use')
+  parser.add_argument('--azure-connection',
+                      help='Name of azure connection to use')
+  parser.add_argument('--aws-access-key', help='S3 access key to use')
+  parser.add_argument('--aws-secret', help='S3 secret access key to use')
+
+  local_args, sccache_args = parser.parse_known_args()
+  return local_args, sccache_args
```

Comment:
Weren't these files deleted in favour of calling `sccache` directly? Why are they needed again?

Reply:
No, it is needed because of how VSTS treats private variables.  They aren't available as environment variables, so we have to pass them in via command line.

Reply:
Resolving because there's another discussion about this on the `${}` vs `$()` comment I made.